### PR TITLE
fix: No search icon on devices from 1024 -> mobile

### LINKED ISSUE #9267: No search icon on devices from 1024 -> mobile
when you resize the screen from 1024 to mobile (480px afaik)
- there is no search icon in the navbar 

image

mobile: 
image

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #9267

## Needs Design check

- @exezbcz please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-02-04 at 08 03 03@2x

### CODE REVIEW

**Review by preschian — APPROVED** [review]

**Comment by exezbcz** [comment]
not sure if its the right solution 

filtering based on touchdevices have some edge cases when the search icon should be there as well 

image

- this is no touch, but smaller window 

the thing is that the search bar is not displayed when the navbar is collapsed - there is then no way how to access the search

**Comment by preschian** [comment]
> this is no touch, but smaller window

@hassnian time to use this instead https://github.com/kodadot/nft-gallery/issues/8767

File: components/Navbar.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
           class="lg:!hidden flex flex-grow items-center justify-end"
           @click="closeBurgerMenu">
           <NeoButton
-            v-if="isMobileNavbarOpen || isTinyMobile"
+            v-if="isMobileNavbarOpen || isTouch"
```
Comment (roiLeo) [inline components/Navbar.vue:21]:
```suggestion
```

this is not needed parent class has already `lg:!hidden`

same form `<Search />` component near it

**Review by roiLeo — CHANGES_REQUESTED** [review]

**Comment by prury** [comment]
as talked with @hassnian on dm, page is now generating two search components, one hidden and another one visible

when searching we also get duplicated hidden/visible results(comparison to canary): 
image
image

solution for tests was to make playwright search only for visible elements within page, although code now a bit more clunky

**Comment by exezbcz** [comment]
@prury better to probably rewrite since the search component does not change based on whether its visible on landing page or not - its always in the navbar 


making it only change to a icon on certain breakpoint would be best solution

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 6.69 USD/DOT ~ 4.484 $DOT
🧗 [16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD](https://polkadot.subscan.io/account/16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)
🔗 [0x80d1d11fb645baaadf96dec35cda1dad5f5c6689b3ec1bfa8c19de5620a2c9e1](https://polkadot.subscan.io/extrinsic/0x80d1d11fb645baaadf96dec35cda1dad5f5c6689b3ec1bfa8c19de5620a2c9e1)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**